### ISSUE TITLE
Monitoring v2 on airgap fail to find shell:v1.0.8 on private registry

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#33995]
**Rancher Server Setup**
- Rancher version: `v2.6.0-rc1`
- Installation option: Single Airgap
   - Airgap, Downstream cluster is RKE1
- Proxy/Cert Details: rancher generated certs

**Information about the Cluster**
- Kubernetes version: `1.21.3`
- Cluster Type: 
   - If downstream, what type of cluster?: Custom RKE1 Airgap

**Describe the bug**

`ImagePullBackOff: Back-off pulling image "<private_registry>/rancher/shell:v0.1.8`

**To Reproduce**

- Setup an Airgap Rancher with Private Registry (basic auth)
- On an airgapped downstream cluster install Monitoring v2 from Dashboard

**Result**

`rancher-monitoring-crd-create` fails to initialize the images.

**Expected Result**

Monitoring v2 installed

**Screenshots**

**Additional context**
`rancher-images.txt` only include `rancher/shell:v0.1.9`

**Comment by cbron** [comment]
@aiyengar2 @jiaqiluo believe we talked about this a while back, can't remember right now situation.

**Comment by prachidamle** [comment]
I believe if `global.cattle.systemDefaultRegistry` is set properly while deploying the chart, then the images will be pulled from the private registry.

Looking at the mentioned error ```ImagePullBackOff: Back-off pulling image "<private_registry>/rancher/shell:v0.1.8```
seems like this value is not getting set?

**Comment by cbron** [comment]
Registry looks right to me: https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-monitoring/rancher-monitoring-crd/100.0.0%2Bup16.6.0/templates/jobs.yaml#L24

Izaac put above: `rancher-images.txt only include rancher/shell:v0.1.9` so that image from CRD chart isn't getting picked up.

**Comment by prachidamle** [comment]
@cbron No I meant the value has to be set by Dashboard - so I was wondering if that is not being set while chart is being deployed

**Comment by kinarashah** [comment]
Available to test with `v2.6.0-rc2`

**Comment by izaac** [comment]
Rancher version: `2.6.0-rc2`
Repo info:

```
spec:
  gitBranch: dev-v2.6
  gitRepo: https://git.rancher.io/charts
status:
  branch: dev-v2.6
  commit: abcc294eed065e8dec10a1757ed9dd5f808d15f4
```

The chart install is now working on an Airgap HA setup with private registry.
